spec/mailers/previews/order_delivery_preview.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class OrderDeliveryPreview < ActionMailer::Preview
+  def delivery_email
+    OrderDeliveryMailer.with(order: order).delivery_email
```

Comment:
is it really working? how are you passing the `order`?

Reply:
yes, this is model defined in factories

Reply:
could you please attach a screenshot of that preview?

Reply:
<img width="424" alt="Screenshot 2022-04-13 at 11 52 53" src="https://user-images.githubusercontent.com/94635848/163152864-50b3ff76-b1ab-4092-9ede-4937d942b190.png">

Reply:
where does the "Cauliflower Penne" comes from?

Reply:
Yes, you are right, fixed: https://github.com/rniedziolka/employee-recognition-platform-1637255604/pull/26/commits/6fb5ecf36a2a620c1b7dfc9a49a73eb67cfd7db1
<img width="780" alt="Screenshot 2022-04-13 at 12 29 34" src="https://user-images.githubusercontent.com/94635848/163159217-2fb4336e-386a-4c51-aaf5-92897197099f.png">